app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -252,9 +252,9 @@ public Calendar getCalendarEndTime() {
     }
 
     public List<WeekViewEvent> toWeekViewEvents(int month) {
-        List<WeekViewEvent> weekViewEvents = new ArrayList<WeekViewEvent>();
+        List<WeekViewEvent> weekViewEvents = new ArrayList<>();
         try {
-            String eventName = String.format("%s %s %s", subject, catalogNumber, type);
+            String eventName = String.format("%s %s", subject, catalogNumber);
```

Comment:
did we decide to show just the course code on the calendar view? we're not going to distinguish the type of course component?